### PR TITLE
Add get option by name

### PR DESCRIPTION
**Pull request by ameech**

### CODE REVIEW

File: src/AbstractCommand.php [hunk 1]
```diff
@@ -31,6 +31,14 @@ public function options()
     /**
      * @inheritDoc
      */
+    public function option($name)
+    {
+        return $this->options[$name];
```
Comment (ameech) [inline src/AbstractCommand.php:36]: What would be acceptable to return here if that option doesn't exist?

File: src/AbstractCommand.php [hunk 2]
```diff
@@ -31,6 +31,14 @@ public function options()
     /**
      * @inheritDoc
      */
+    public function option($name)
```
Comment (shadowhand) [inline src/AbstractCommand.php:34]: Not sure why you need this... once you pass `$this->options()` than all required stuff is known to be defined.

If anything, this points to the need for #6

Comment (ameech) [inline src/AbstractCommand.php:34]: Just an easier to way to get an option vs `$this->options()['example']` or

``` PHP
$options = $this->options();
$example = $options['example'];
```

Comment (shadowhand) [inline src/AbstractCommand.php:34]: Then yeah, I think #6 is what we want.

Comment (elazar) [inline src/AbstractCommand.php:34]: @shadowhand As in close #7 in favor of #6?

Comment (shadowhand) [inline src/AbstractCommand.php:34]: Yes I think so.

**Pull request closed without merging**